src/js/course.ts

```diff
@@ -470,6 +491,17 @@ export function isClosed(c: CourseV3) {
   return c.courseEnrollmentStatus === "closed";
 }
 
+export function getCourseArea(c: CourseV3): CourseArea {
+  const code = c.courseCode.split(" ")[0];
+  if (code === "PE") {
+    return CourseArea.Pe;
+  } else if (filterTechs.includes(code)) {
+    return CourseArea.Tech;
+  } else {
+    return CourseArea.Hum;
+  }
```

Comment:
This is short enough that I'm inclined to prefer a conditional/tertiary _expression_ here instead:
```typescript
code === 'PE' ? CourseArea.Pe
  : filterTechs.includes(code) ? CourseArea.Tech : CourseArea.Hum;
```